CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.11)
+cmake_minimum_required (VERSION 3.22)
```

Comment:
This is dangerous: Cmake v3.22 is younger than 3 months and almost everyone will run into build problems! FU has v3.18 on their servers, MPI has v3.16. Which functions of v3.22 are you actually using?